NEWS.md

```diff
@@ -1,5 +1,8 @@
 # tidyselect (development version)
 
+* New `tidyselect_proxy_data()` and `tidyselect_proxy_supports_predicates()`
```

Comment:
Any ideas for better names? I have now mistyped them 3+ times.

Reply:
I'd be fine with dropping `data`

```r
tidyselect_proxy(data)
tidyselect_supports_predicates(data)
```

It feels slightly redundant since `data` is the argument name too.

And I'm not too worried about it ever being ambiguous with anything else in tidyselect